app/src/main/res/layout/fragment_publish.xml

```diff
@@ -6,18 +6,376 @@
     android:layout_height="match_parent"
     tools:context=".ui.publish.PublishFragment">
 
-    <TextView
-        android:id="@+id/text_dashboard"
-        android:layout_width="match_parent"
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/publishExperimentButton"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="8dp"
-        android:layout_marginTop="8dp"
-        android:layout_marginEnd="8dp"
-        android:textAlignment="center"
-        android:textSize="20sp"
+        android:layout_marginBottom="72dp"
+        android:text="Publish"
+        app:backgroundTint="#8BC34A"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/experimentVariableNameLayout"

```

Comment:
Based on our coding conventions, I think the id should be something like `experiment_variable_name_layout`

Same for the other ids.

Reply:
Changed it. Also removed the experiment title input.